src/engine/e_bluefield.c

```diff
@@ -45,24 +48,35 @@ static int engine_pka_init(ENGINE *e);
 static int engine_pka_finish(ENGINE *e);
 void engine_load_pka_int(void);
 
-/* BN operations */
 /* BN mod_exp */
 static int engine_pka_bn_mod_exp(BIGNUM *r, const BIGNUM *a, const BIGNUM *p,
                                  const BIGNUM *m, BN_CTX *ctx, BN_MONT_CTX *m_ctx);
-/* BN mod_inv */
-static int
-engine_pka_bn_mod_inv(const EC_GROUP *group, BIGNUM *r, const BIGNUM *x,
-                      BN_CTX *ctx);
 
-# if (OPENSSL_VERSION_NUMBER >= 0x10100000L)
```

Comment:
Any reason why we removed those guards?

Reply:
It is still valid, isn't? my only concern is when customers install an unsupported OpenSSL version, are they able to build the PKA software. I believe it is safe/harmless to leave such define, am I wrong?

Reply:
Oops, never mind. I thought this define is for OpenSSL 1.0.0